contrib/compose/nginx/Dockerfile

```diff
@@ -0,0 +1,6 @@
+FROM nginx
```

Comment:
Perhaps this should be pinned to a specific version, e.g. `nginx:1.7` for "best practice"?